### PR TITLE
Added 3 Anime's from animeultima.eu and correcting one typo

### PR DESCRIPTION
**Pull request by akuntamvan**
- [x] 🔍 I have searched the `data.js` file and confirmed I am not adding a duplicate entry. Note: Different season of the same anime are okay to add such as Sword art online (2013) and Sword art online (2014) .
- [x] 🌻 I have added a anime year under `animeYear`. Note: Do not add ranges such as 2017-2019.
- [x] 🔗 I have added the myanimelist page under `animeLink`.
- [x] 3️⃣ I have at least three emojis listed under `animeEmoji`.
- [x] 5️⃣ I have a maximum of five emojis listed under `animeEmoji`.
- [x] 👍 My pull request has a descriptive title (such as `Added Sword Art Online` or `Added Sword Art Online: Alicization`).
- [x] 🖍️ I have placed the new show(s) or movie(s) in alphabetical order based on title. If the show or movie starts with 'the', then use the second word to alphabetize.

### CODE REVIEW

**Comment by YogaSakti** [comment]
thank you for contributing

File: src/store/listAnime.js [hunk 1]
```diff
@@ -217,7 +217,7 @@ export default [
     },
     {
         animeEmoji:"🍌🐱",
-        animeName:"Banya: Fushigi na Nakama-tachi",
+        animeName:"Bananya: Fushigi na Nakama-tachi",
         animeYear:2019,
         animeLink:"https://www15.animeultima.eu/a/bananya-fushigi-na-nakama-tachi_379802"
```
Comment (2pai) [inline src/store/listAnime.js]: can you use myanimelist link?

**Comment by YogaSakti** [comment]
and at least three emojis listed under animeEmoji

File: src/store/listAnime.js (hunk 1 above)
Comment (akuntamvan) [reply src/store/listAnime.js]: oh my bad. I'll do it now.

**Pull request closed without merging**